settings.html

```diff
@@ -35,48 +36,39 @@ <h2 style="font-family: Gill Sans Light">Auto-archive Settings</h2>
-                    <h2 style="font-family: Gill Sans Light">Amazon URLs</h2>
-                    <h4>Read book from archive.org</h4>
+                    <h2 style="font-family: Gill Sans Light">Amazon</h2>
                     <label for="books" style="font-size: 14px;">
                         <input type="checkbox" id="books">
-                        <span class="enable"><b>Show "Read This Book" button</b></span>
-                    </label>
-                </div>
-            </div>
-            <div id="show_option_news"  class="col-sm-3">
-                <div class="news" style="padding-top: 5px;">
-                    <h2 style="font-family: Gill Sans Light">Show News Recommendation</h2>
-                    <h4>Show Web News -> TV News Recommendations</h4>
-                    <label for="news" style="font-size: 14px;">
-                        <input type="checkbox" id="news">
-                        <span class="enable"><b>Find TV News Clips</b></span>
+                        <span class="enable"><b>Enable "Read This Book" button</b></span>
                     </label>
                 </div>
             </div>
 						<div id="show_option_wikibooks"  class="col-sm-3">
                 <div class="wikibooks" style="padding-top: 5px;">
-                    <h2 style="font-family: Gill Sans Light">Wikipedia books settings</h2>
-                    <h4>Read books cited by Wikipedia</h4>
+                    <h2 style="font-family: Gill Sans Light">Wikipedia</h2>
                     <label for="wikibooks" style="font-size: 14px;">
                         <input type="checkbox" id="wikibooks">
-                        <span class="enable"><b>Find Books</b></span>
-                    </label>
-                </div>
-            </div>
-						<div id="show_option_doi"  class="col-sm-3">
-                <div class="doi" style="padding-top: 5px;">
-                    <h2 style="font-family: Gill Sans Light">Wikipedia papers settings</h2>
-                    <h4>Read papers cited by Wikipedia</h4>
-                    <label for="doi" style="font-size: 14px;">
-                        <input type="checkbox" id="doi">
-                        <span class="enable"><b>Find Papers</b></span>
+                        <span class="enable"><b>Find Cited Books</b></span>
                     </label>
+										<br>
+										<label for="doi" style="font-size: 14px;">
+												<input type="checkbox" id="doi">
+												<span class="enable"><b>Find Cited Papers</b></span>
+										</label>
                 </div>
             </div>
+						<div id="show_option_news"  class="col-sm-3">
+								<div class="news" style="padding-top: 5px;">
+										<h2 style="font-family: Gill Sans Light">News Articles</h2>
+										<label for="news" style="font-size: 14px;">
+												<input type="checkbox" id="news">
+												<span class="enable"><b>Find TV News Clips</b></span>
+										</label>
+								</div>
+						</div>
```

Comment:
Hey, what's wrong with this crazy spacing? :)

Reply:
Looks fine on atom. I'll see if it changes if I push again